packages/arista_ngfw/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,826 @@
+      target_field: user.name
+      ignore_missing: true
+
+  #########################
+  ## ECS Related Mapping ##
+  #########################
+  # Hosts
+  - append:
+      if: ctx.destination?.domain != null
+      field: related.hosts
+      value: '{{{destination.domain}}}'
+      allow_duplicates: false
+  - append:
+      if: ctx.source?.domain != null
+      field: related.hosts
+      value: '{{{source.domain}}}'
+      allow_duplicates: false
+  # IP Addresses
+  - append:
+      if: ctx.destination?.ip != null
+      field: related.ip
+      value: '{{{destination.ip}}}'
+      allow_duplicates: false
+  - append:
+      if: ctx.destination?.nat?.ip != null
+      field: related.ip
+      value: '{{{destination.nat.ip}}}'
+      allow_duplicates: false
+  - append:
+      if: ctx.host?.ip != null
+      field: related.ip
+      value: '{{{host.ip}}}'
+      allow_duplicates: false
+  - append:
+      if: ctx.source?.ip != null
+      field: related.ip
+      value: '{{{source.ip}}}'
+      allow_duplicates: false
+  - append:
+      if: ctx.source?.nat?.ip != null
+      field: related.ip
+      value: '{{{source.nat.ip}}}'
+      allow_duplicates: false
+  # Users
+  - append:
+      if: ctx.user?.name != null
+      field: related.user
+      value: '{{{user.name}}}'
+      allow_duplicates: false
+
+  #################################
+  ## ECS Arista-Specific Mapping ##
+  #################################
+  - rename:
+      field: arista.policyId
+      target_field: arista.policy.id
+      ignore_missing: true
+  - rename:
+      field: arista.policyRuleId
+      target_field: arista.policy.rule_id
+      ignore_missing: true
+  
+  ##############
+  ## Clean Up ##
+  ##############
+  - remove:
+      description: Remove Arista fields that are not necessary after parsing to ECS
+      field:
+        - arista.blocked
+        - arista.categoryId
+        - arista.class
+        - arista.clientCountry
+        - arista.clientLatitude
+        - arista.clientLongitude
+        - arista.filterPrefix
+        - arista.hostname
+        - arista.localAddr
+        - arista.remoteAddr
+        - arista.httpRequestEvent.sessionEvent
+        - arista.requestLine
+        - arista.serverCountry
+        - arista.serverLatitude
+        - arista.serverLongitude
+        - arista.sessionEvent
+        - arista.tagsString
+        - arista.timeStamp
+      ignore_missing: true
+  - remove:
+      description: Remove Arista root field if empty
+      field: arista
+      if: ctx.arista != null && ctx['arista'].empty
+  - remove:
+      field: event.original
+      if: "ctx?.tags == null || !(ctx.tags.contains('preserve_original_event'))"
+      ignore_failure: true
+      ignore_missing: true
+  - remove:
+      field:
+        - _temp
+        - _temp_
+        - _conf
+        - _ingest
+      ignore_missing: true
+on_failure:
+  - set:
+      field: error.message
+      value: '{{ _ingest.on_failure_message }}'
```

Let's make this
```suggestion
  - set:
      field: event.kind
      value: pipeline_error
  - append:
      field: error.message
      value: "{{ _ingest.on_failure_message }}"
```
similar for all the other sub-pipelines.

---

Forward thinking for [what we've discussed](https://github.com/elastic/integrations/pull/6258#issuecomment-1575749979) about the use of `error.message` for errors in the actual events. I like it! :-D